src/pattern-library/patterns/labels/labels.md

```diff
@@ -1 +1,4 @@
-Labels can be used to display a filename associated with a [code](/handbook/markup-code) snippet.
+Labels can be used to display a filename associated with a [code]
```

Comment:
I think it's because there is a newline between [code] and (/handbook/...)
![image](https://user-images.githubusercontent.com/1066253/130166780-195eed59-94e0-4794-80b2-04d267425b6f.png). I can add a suggestion to fix it.

Reply:
Ah this is Vim playing sill buggers. I use `gq` to shrink the characters per line and it looks like it messed up there. Good spot!